services/placement/placement.go

```diff
@@ -80,12 +80,20 @@ func (p placement) NumShards() int {
 	return len(p.shards)
 }
 
+func (p placement) Sharded() bool {
+	return p.rf != 0
```

Comment:
Using replica factor to indicate whether the placement is sharded is strange, in that non-sharded services essentially have one copy of the entire dataset. Can we add a boolean to indicate whether the placement is sharded or not?